site/docs/v1/tech/apis/authentication.adoc

```diff
@@ -6,124 +6,152 @@ description: Learn how the FusionAuth APIs authenticate requests
 
-For example, the User API `/api/user` has four HTTP methods, `GET`, `POST`, `PUT` and `DELETE`. While each API may have different semantics, in a general sense you can think of these four HTTP methods as being retrieve, create, update and delete respectively. With that in mind, if you'd like to create an API key that can only retrieve users, you would limit the API key to the `GET` method on the `/api/user` API.
+For example, the User API `/api/user` has five HTTP methods, `GET`, `POST`, `PUT`, `PATCH` and `DELETE`. While each API may have different semantics, in a general sense you can think of these HTTP methods as being retrieve, create, update, partial update, and delete respectively. With that in mind, if you'd like to create an API key that can only retrieve users, limit the API key to the `GET` method on the `/api/user` API.
+
+When you create an API key, the key value is defaulted to a secure random value. However, the API key is simply a string, so you may set the key value to `super-secret-key` if you'd like. A long and random value makes a good API key in that it is unique and difficult to guess, so allowing FusionAuth to create the key value is recommended.
 
-When you create an API key the key is defaulted to a secure random value but the API key is simply a string, so you may call it `super-secret-key` if you'd like. However a long and random value makes a good API key in that it is unique and difficult to guess.
+=== Managing API Keys via the API
+
+Prior to version 1.26.0, the FusionAuth administrative user interface was the only way to create API keys. This functionality was not available through an API. Starting from version 1.26.0, API keys may be created using an API. Please refer to the link:/docs/v1/tech/apis/api-keys/[API Key API] for more information.
 
 === Create an API Key
 
-image::create-api-key.png[Create an API Key,width=1200,role=shadowed]
+image::create-api-key.png[Create an API Key,width=1200]
 
 ==== Form Fields
 
 [.api]
+[field]#Id#::
+The unique Id of this API key.
+
 [field]#Key#::
-A unique key to be utilized to authorize API requests using the `Authorization` header.
+The unique string representing the API key. This is what is presented in the `Authorization` header for requests to FusionAuth.
 
 [field]#Description#::
-An optional description of how this API key will be utilized.
+An optional description of this API key.
 
 [field]#Tenant#::
-The optional tenant to which this API key will be assigned. This value cannot be changed once the API key is created. This field is only displayed when more than one tenant exists. When you assign an API key to a tenant, you will only be able to operate on users, applications, and groups in the selected tenant.
+The optional tenant to which this API key will be assigned. This value cannot be changed once the API key is created. This field is only displayed when more than one tenant exists in the FusionAuth instance. 
```

Comment:
I may have changed this in the last release, I think it is always displayed and can be set even when a single tenant exists. This was based upon user feedback. You can double check if you want, I think it is always displayed, and selectable always as well.

Reply:
Double checked. The Tenant dropdown appears all the time now, but can't be modified after a key is created.